considerations.md

```diff
@@ -5,7 +5,7 @@ Instead they MUST ignore unknown properties.
 
 # Canonicalization
 
-OCI Images [are](descriptor.md) [content-addressable](image-layout.md).
+OCI Images are content-addressible. See [Descriptors](descriptor.md) and [Image Layout](image-layout.md) for more information.
```

Comment:
Hmm.  While `descriptor.md` mentions Merkle DAGs, it doesn't say "content addressable".  I think we need to update @jonboulle's suggestion to  keep the `image-layout.md` link.